proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/structure/MetsElementVisitor.java

```diff
@@ -854,6 +855,8 @@ private void addPremisToAmdSec(AmdSecType amdSec, HashMap<String, FileMD5Info> m
         toGenerate.put("OBJ_001", "RAW");
         toGenerate.put("OBJ_002", "MC_IMGGRP");
         toGenerate.put("OBJ_003", "ALTOGRP");
+        toGenerate.put("OBJ_004", "UC_IMGGRP");
```

Comment:
Tohle by mělo být definováno jako konstanty. Hodnoty jsou použité na spoustě míst.